### ISSUE TITLE
Hook failed when using typescript

### ISSUE THREAD

**Issue by shcode** [buefy/vue-cli-plugin-buefy#10]
I try to add buefy to vue js project which using typescript. Get this error:

```
⠋  Running completion hooks... ERROR  Error: ENOENT: no such file or directory, open '/Users/shcode/Projects/playground/vue/{project}/src/main.js'
Error: ENOENT: no such file or directory, open '/Users/shcode/Projects/playground/vue/{project}/src/main.js'
    at Object.fs.openSync (fs.js:646:18)
    at Object.fs.readFileSync (fs.js:551:33)
    at api.onCreateComplete (/Users/shcode/Projects/playground/vue/{project}/node_modules/vue-cli-plugin-buefy/generator/index.js:28:26)
    at runGenerator (/Users/shcode/.config/yarn/global/node_modules/@vue/cli/lib/invoke.js:136:13)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
```

**Comment by gonzalobustos** [comment]
I am experiencing the exact same issue...

I am using Vue CLI v3.0.0-rc.8 and the steps to reproduce are:

```
vue create my-project (vue-router, vuex, sass, babel, typescript, pwa, unit-jest, e2e-nightwatch)
vue add buefy
```

**Comment by anteriovieira** [comment]
Fixed in version `v0.3.0` :smiley:
